test/build-tracker.test.js

```diff
@@ -0,0 +1,82 @@
+const assert = require('assert')
+const {describe, it, before} = require('mocha')
+
+const Tracker = require('../src/trackers/classes/tracker')
+const Rule = require('../src/trackers/classes/rule')
+const sharedData = require('../src/trackers/helpers/sharedData')
+
+const commonRequestData1 = {
+    apis: {},
+    cnames: [],
+    cookies: 1,
+    fpAvg: 0,
+    fpStd: 0,
+    host: "tracker.com",
+    prevalence: 1,
+    responseHashes: [],
+    rule: "tracker\\.com\\/collect",
+    sites: 2,
+    subdomains: new Set(['dummy']),
+    type: "XHR",
+}
+const commonRequestData2 = {
```

Comment:
Do these need `nameservers` since I just added those?

Reply:
This test doesn't currently test all attributes, but we could add tests for nameservers fairly easily here. However, to properly test the fetching of nameserver data we'd have to mock the ns lookups, as relying on real lookups in tests is not a great idea.

Reply:
Makes sense, thanks.